### PR TITLE
[ADD] general: debug doc + link to the file on mentions over other apps

### PR DESCRIPTION
**Pull request by kbr-odoo**
I'll look for more mentions on user docs once at least a few of the ones waiting to be merged, are merged.

For the developer documentation: will do it after your instructions. I'll message you.

### CODE REVIEW

**Comment by kbr-odoo** [comment]
I'll look for more mentions once at least a few of these docs waiting to be merged, are merged.

**Comment by AntoineVDV** [comment]
> I'll look for more mentions on user docs once at least a few of the ones waiting to be merged, are merged.

First merged wins! You only care about what's in your branch as long as it's not merged. After that, it's no more your concern.
In other words: right after that your branch is merged, all references in 13.0 to the debug mode should point to your doc. All pending branches (PRs) will have to rebase on 13.0 (and thus on your commit) to include your page and correctly reference it.

> For the developer documentation: will do it after your instructions. I'll message you.

For the dev doc we can wait that this PR is merged, no hurry :)

**Comment by kbr-odoo** [comment]
Aye aye, captain!

File: accounting/localizations/france.rst [hunk 1]
```diff
@@ -155,10 +155,9 @@ Invoicing and Accounting apps.
    If you record a new sales transaction for a period already closed, 
    it will be counted in the very next closing.
 
-.. tip:: For test & audit purposes such closings can be manually generated in the 
- developer mode. Go to 
- :menuselection:`Settings --> Technical --> Automation --> Scheduled Actions` 
- to do so.
+.. tip:: For test & audit purposes such closings can be manually generated in the
+   :doc:`Developer mode <../../general/debug/debug>`. Then go to
+   :menuselection:`Settings --> Technical --> Automation --> Scheduled Actions`.
```
Comment (AntoineVDV) [inline accounting/localizations/france.rst]:
```suggestion
.. tip:: For test & audit purposes such closings can be manually generated with the
   :doc:`Developer mode <../../general/debug/debug>`. Enable it and then go to
   :menuselection:`Settings --> Technical --> Automation --> Scheduled Actions`.
```

File: accounting/localizations/mexico.rst [hunk 2]
```diff
@@ -106,10 +106,7 @@ Enabling CFDI Version 3.3
    SaaS instance please ask for an upgrade by submitting a ticket to support in
    https://www.odoo.com/help.
 
-Enable debug mode:
-
-.. image:: media/mexico10.png
-   :align: center
+Enable :doc:`Developer mode <../../general/debug/debug>`.
```
Comment (AntoineVDV) [inline accounting/localizations/mexico.rst]:
```suggestion
Enable the :doc:`Developer mode <../../general/debug/debug>`.
```

File: discuss/email_servers.rst [hunk 3]
```diff
@@ -159,12 +159,9 @@ alias in your mail server.
...
- developer mode from the Settings Dashboard.
-
- .. image:: media/developer_mode.png
-    :align: center
+   Object aliases can be edited from their  respective configuration view.
+   To edit catchall and bounce aliases, you first need to activate the
+   :doc:`Developer mode <../../general/debug/debug>`.
 
  Then refresh your screen and go to 
  :menuselection:`Settings --> Technical --> Parameters --> System Parameters`
```
Comment (AntoineVDV) [inline discuss/email_servers.rst]: Enabling the debug mode already refresh the page.
```suggestion
 Then go to :menuselection:`Settings --> Technical --> Parameters --> System Parameters` 
```

File: discuss/email_servers.rst [hunk 4]
```diff
@@ -174,7 +171,7 @@ alias in your mail server.
     :align: center
 
 .. note:: By default inbound messages are fetched every 5 minutes in Odoo on-premise. 
-   You can change this value in developer mode.
+   You can change this value in :doc:`Developer mode <../../general/debug/debug>`.
    Go to :menuselection:`Settings --> Technical --> Automation --> 
    Scheduled Actions` and look for *Mail: Fetchmail Service*.
```
Comment (AntoineVDV) [inline discuss/email_servers.rst]:
```suggestion
   You can change this value with the :doc:`Developer mode <../../general/debug/debug>`. Enable it
   and go to :menuselection:`Settings --> Technical --> Automation --> Scheduled Actions` and look
   for *Mail: Fetchmail Service*.
```

File: general/in_app_purchase/in_app_purchase.rst [hunk 5]
```diff
@@ -24,8 +24,8 @@ IAP accounts
 
 Credits to use IAP services are stored on IAP accounts, which are specific to each service and
 database. By default, IAP accounts are common to all companies, but can be restricted to specific
-ones by going to :menuselection:`Settings app --> Activate the Developer Mode --> Technical Settings
---> IAP Account`.
+ones. Activate the :doc:`Developer mode <../../general/debug/debug>` then go to
```
Comment (AntoineVDV) [inline general/in_app_purchase/in_app_purchase.rst]:
```suggestion
ones. Activate the :doc:`Developer mode <../../general/debug/debug>`, then go to
```

File: accounting/bank/feeds/bank_synchronization.rst [hunk 6]
```diff
@@ -121,8 +121,7 @@ database, please `submit a support ticket <https://www.odoo.com/help>`_.
...
 How can I update my bank credentials?
 -------------------------------------
-You can update your credentials in the *Developer Mode*.
+You can update your credentials in the :doc:`Developer mode <../../../general/debug/debug>`.
 
-To do so, go to :menuselection:`General Settings`, click on *Activate developer mode*, then go to
-:menuselection:`Accounting --> Configuration --> Online Synchronization`, then open the Institution
-you want to edit, and click on *Update Credentials*.
\ No newline at end of file
+Then go to :menuselection:`Accounting --> Configuration --> Online Synchronization`, then open the
```
Comment (AntoineVDV) [inline accounting/bank/feeds/bank_synchronization.rst]:
```suggestion
Go to :menuselection:`Accounting --> Configuration --> Online Synchronization`, then open the
```

**Pull request merged**